frontend/docs/admin-guide/install.md

```diff
@@ -1087,11 +1073,6 @@ policy. From the IAM Service page click "Policies" on the left hand side.
             "Resource": "arn:{AWS_PARTITION}:elasticmapreduce:{AWS_REGION}:{AWS_ACCOUNT}:cluster/*",
             "Effect": "Allow"
         },
-        {
-            "Action": "comprehend:DetectDominantLanguage",
```

Comment:
I don't see this in this policy anymore. Do you happen to know if it went away completely and/or why?

Reply:
Yeah, sorry I should have commented. This is covered by the notebook policy.
- [IAM stack](https://github.com/awslabs/mlspace/blob/main/lib/stacks/iam.ts#L67)
- [Docs](https://github.com/awslabs/mlspace/blob/main/frontend/docs/admin-guide/install.md?plain=1#L308)